arrayfire/array.py

```diff
@@ -1185,20 +1185,35 @@ def to_list(self, row_major=False):
         ct_array, shape = self.to_ctype(row_major, True)
         return _ctype_to_lists(ct_array, len(shape) - 1, shape)
 
-    def __repr__(self):
+    def to_string(self):
```

Comment:
It should in a perfect world, where IDEs only use `__repr__` for variable representation. Unfortunately that's not the case, PyCharm displays variables by `__repr__`, but calls `__str__` to for an unknown reason. So, even `__repr__` got faster implementation, PyCharm stays as a hog, with unusable debugger. So, I decided to implement the slow to string method in an unstandard way, to prevent *smart* IDEs to get braincock by calling it accidentally.